pkg/virt-handler/cgroup/util.go

```diff
@@ -129,6 +129,19 @@ func generateDeviceRulesForVMI(vmi *v1.VirtualMachineInstance, isolationRes isol
 			vmiDeviceRules = append(vmiDeviceRules, deviceRule)
 		}
 	}
+	if vmi.Spec.Domain.Devices.Rng != nil {
```

Comment:
Maybe we should include it by default?

Reply:
So whitelist it even when there's no need? why not give out precisely what's requested, since we can easily tell?

Reply:
Btw, what is the bug we observed?

Reply:
When running in cgroupsv2, and hotplugging a block disk, and having an RNG device, you are unable to live migrate due to the rng device permissions disappearing.